packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.spec.tsx

```diff
@@ -209,350 +207,354 @@ describe("viewport positioner", (): void => {
     });
 
     test("positioning values applied correctly for specified default position - adjacent + top + left", (): void => {
-        const anchorElement: React.RefObject<HTMLDivElement> = React.createRef<
-            HTMLDivElement
-        >();
+        const container: HTMLDivElement = document.createElement("div");
+        document.body.appendChild(container);
 
         const rendered: any = mount(
-            <div>
-                <div ref={anchorElement} />
-                <ViewportPositioner
-                    viewport={document.firstElementChild as HTMLElement}
-                    horizontalPositioningMode={AxisPositioningMode.adjacent}
-                    defaultHorizontalPosition={ViewportPositionerHorizontalPosition.left}
-                    verticalPositioningMode={AxisPositioningMode.adjacent}
-                    defaultVerticalPosition={ViewportPositionerVerticalPosition.top}
-                    anchor={anchorElement}
-                    managedClasses={managedClasses}
-                />
-            </div>
-        );
-
-        const positioner: any = rendered.find("BaseViewportPositioner");
-        positioner.instance().updateLayout();
-        expect(positioner.instance().state.currentHorizontalPosition).toBe(
+            <ViewportPositioner
+                viewport={container}
+                horizontalPositioningMode={AxisPositioningMode.adjacent}
+                defaultHorizontalPosition={ViewportPositionerHorizontalPosition.left}
+                verticalPositioningMode={AxisPositioningMode.adjacent}
+                defaultVerticalPosition={ViewportPositionerVerticalPosition.top}
+                anchor={anchorElement}
+                verticalLockToDefault={true}
+                horizontalLockToDefault={true}
+                managedClasses={managedClasses}
+            />,
+            { attachTo: container }
```

Comment:
This is starting to look like it's potentially changing implementation of this component. Can you verify the existing method of using a react ref still works?

Reply:
We recently removed the extract ref function from this component and moved it to a shared utility and we test extraction there (utilities/extract-element.spec.tsx):

 test("extractHtmlElement function returns element passed in as a ref", (): void => {
        const rendered: any = shallow(<TestClass />);

        const extracted: HTMLElement = extractHtmlElement(rendered.instance().testRef);
        expect(extracted).toEqual(rendered.instance().testRef.current);
    });

At that time I moved the tests for that function out of this spec file and into that one, as well adding the test above.  I can duplicate something like that here if desired, but seems somewhat redundant to me?  I could go either way.